packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js

```diff
@@ -40,8 +40,13 @@ export default class QuanticResult extends LightningElement {
 
   @track resultHasPreview = true;
 
+  /** @type {boolean} */
+  isHovered = false;
+
   connectedCallback() {
     this.template.addEventListener('haspreview', this.onHasPreview);
+    this.template.host.addEventListener('mouseenter', this.setHoverState);
```

Comment:
I guess it is out of scope, but what about keyboard navigation?

Reply:
Yea I think we could switch to something like `isFocused` instead and support maybe the keyboard navigation?

Don't have to support keyboard for this PR, but we could already use the name `focused` instead of `hovered` for clarity as well I think I prefer that name, what about you all?

Reply:
Hover and Focus are two different things though. But I agree we should ultimately offer both. For example, the result actions should display both on hover and on focus.

Reply:
I agree with Nathan, sometimes we want two different behaviours on hover and on focus since they are two separate things. We now have the `isHovered` property and I'll create a new ticket to add the `isFocused` property, the property that will allow us to support the keyboard navigation.

Reply:
I can't find a single term that conveys the idea of `focused or hovered`. Perhaps `focused` is the best one.